feegrant/index.md

```diff
@@ -0,0 +1,239 @@
+---
+parent:
+  title: Understanding the Fee Grant Module
+order: 0
+description: Use the Cosmos SDK feegrant module to grant the use of tokens to pay for fees from one account (the granter) to another account (the grantee).
+---
+
+# Feegrant Module
+
+The [feegrant](https://docs.cosmos.network/v0.44/modules/feegrant/) enables a user / contract / module (the granter) to pay the fees for someone else (the grantee) when they want to broadcast a transaction on the blockchain. A key feature of feegrant is that the granter retains full access to their tokens, and is able to revoke the allowance at any time.
+
+## Use Feegrant to Grant an Allowance
+
+An often discussed use case for the feegrant module is that blockchains or smart contracts can allow new users to start interacting with the chain or contract without having to acquire tokens first, greatly improving the onboarding experience.
+
+There are two [fee allowance types](https://docs.cosmos.network/v0.44/modules/feegrant/01_concepts.html#fee-allowance-types) that the feegrant module implements: `BasicAllowance` and `PeriodicAllowance`, the latter allows you to setup a periodic allowance to another user. In this tutorial, you will set up two tokens in your blockchain: one default token called `stake`, which will be used for fees, and another called `kudos`, for sending to your friends.
```

Comment:
```suggestion
An often discussed use case for the fee grant module is improved onboarding experience because new users don't have to acquire tokens before they can start interacting with the blockchain or smart contract.  

Two [fee allowance types](https://docs.cosmos.network/v0.44/modules/feegrant/01_concepts.html#fee-allowance-types) are implemented with the fee grant module:

- `BasicAllowance`

    Grantee uses fees from a granter's account.

- `PeriodicAllowance`

    A periodic allowance to another user. 

In this tutorial, you will set up two tokens in your blockchain: a default token called `stake` to use for fees and another token called `kudos` to send to your friends.
```

Reply:
let's see if using list items works here for the two allowance types.